docs/BEST_PRACTICES.md

```diff
@@ -1,4 +1,17 @@
-## Deployment best practices
+## Deployment Guide
+
+As a quick start, you can utilize the available bash script which is capable of deploying both modules to a 
+Google Cloud Project. It requires to have an existing Kubernates cluster in Google Kubernates Engine. 
+Edit the configuration values in  deploy-to-gcp.sh and execute it.
+
+| Name | Description | Required | Default value (only for test environment) |
```

Comment:
What about passing arguments to script? or you might describe the example of how to run it after the table.

Reply:
It would be much visible to have it on the script it-self. If someone wants to make to ask parameters, it's a subtle change. c3293f785c5f2d7bc23617fb8238a97af0c75594